payloads/v8/_interactions/responses.ts

```diff
@@ -76,3 +87,7 @@ export type APIInteractionResponseCallbackData = Omit<
 	RESTPostAPIWebhookWithTokenJSONBody,
 	'username' | 'avatar_url'
 > & { flags?: MessageFlags };
+
+export interface APICommandAutocompleteInteractionResponseCallbackData extends APIInteractionResponseCallbackData {
```

Comment:
No..thats also not the interface / response this type expects... It only expects the choices array 😓